map.py

```diff
@@ -0,0 +1,47 @@
+from point import Point
+import random
+import itertools
+
+class Map:
+    def __init__(self, num_of_points, map_size, seed):
+        self.num_of_points = num_of_points
+        self.map_size = map_size
+        self.seed = seed
+        self.points = set()
+
+    def generate_map(self):
+        """
+        A function that generates random points with their coordinates and limits
+        with Map objet's parametars
+        
+        """
+        random.seed(self.seed)
+        for i in range(self.num_of_points):
+            point = Point(random.uniform(0, self.map_size), random.uniform(0, self.map_size))
+            if point in self.points:
+                i -= 1
```

Comment:
Ovo mislim da ne radi bas tako. Ako se ne varam funkcija `range` generise listu i onda iterira po njoj. Sto bi znacilo da ako smanjis `i` za jedan samo ces mu promeniti vrednost iteratora a neces pomeriti iterator nazad za jedan

Reply:
upravu si, ostavio sam to iz prosle. Promenjeno u while